code/datums/mind.dm

```diff
@@ -1038,27 +1038,15 @@
 		switch(href_list["wizard"])
 			if("clear")
 				remove_wizard()
-				to_chat(current, "<span class='userdanger'>You have been brainwashed! You are no longer a wizard!</span>")
 				log_admin("[key_name(usr)] has de-wizard'ed [current].")
-				SSticker.mode.update_wiz_icons_removed(src)
 			if("wizard")
-				if(!(src in SSticker.mode.wizards))
-					SSticker.mode.wizards += src
+				if(has_antag_datum(/datum/antagonist/wizard))
 					special_role = "Wizard"
-					//SSticker.mode.learn_basic_spells(current)
-					to_chat(current, "<span class='boldannounce'>You are the Space Wizard!</span>")
+					add_antag_datum(/datum/antagonist/wizard)
 					message_admins("[key_name_admin(usr)] has wizard'ed [current].")
 					log_admin("[key_name(usr)] has wizard'ed [current].")
-					SSticker.mode.update_wiz_icons_added(src)
 			if("lair")
 				current.forceMove(pick(GLOB.wizardstart))
-			if("dressup")
-				SSticker.mode.equip_wizard(current)

```

Comment:
Is there a reason dressup & rename is being removed from the TP?

Reply:
Rename is redundant since standard mob rename achieves the same, and there's really no point to duplicate it on traitor panel.  Same with dressup you can select proper outfit if you want to do this manually. No need to introduce five ways of doing the same thing really. I can re-add it but i doubt it was much used function.